### PR TITLE
Desktop version done with animation

### PR DESCRIPTION
**Pull request by BilalLiaquat7**
In this PR: Desktop version of project done
change in style file
change in HTML file
svg file(s) added
animation implemented

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -365,3 +365,166 @@ h5 {
     }
   }
 }
+
+@media only screen and (min-width: 768px) {
```
Comment (christianbiring1) [inline style.css:369]: - [ ] Kindly make sure on the screen size of `768px` tablet your elements are well positioned as you may notice how your page looks messy. Please make sure you stick to the original template provided in Figma.
Hint: You may consider using the flexbox and the grid container to display your layout.

Screenshot from 2023-06-15 15-31-09
Screenshot from 2023-06-15 15-37-23
Screenshot from 2023-06-15 15-38-18
Screenshot from 2023-06-15 15-41-32

image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @BilalLiaquat7 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 


## Highlights
- [ ] Great job working on the desktop version of your portfolio. 
- [ ] Very descriptive PR `title` and `summary` :+1: 
- [ ] All linters checks are passing :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -16,31 +16,38 @@
...
             <a href="#"><h2>Bilal</h2></a>
         </div>
-        <nav class="hamBurger">
+        <nav class="Menuicon">
             <span class="material-symbols-outlined">
                 menu
                 </span>
         </nav>
+        <ul class="menu">
+            <li><a class="menu_item" href="#portfolio">portfolio</a></li>
```
Comment (franclobo) [inline index.html:25]: - **[OPTIONAL]** Kindly use capital letters for the "Portfolio" link to be similar to the Figma template.

**Review by franclobo — APPROVED** [review]
Hi @BilalLiaquat7 and @duaneandrea, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Desktop version for the width screen 1440px looks similar to the Figma template ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**